deps/wxWidgets/wxWidgets.cmake

```diff
@@ -49,6 +49,7 @@ bambustudio_add_cmake_project(wxWidgets
         -DwxUSE_LIBJPEG=sys
         -DwxUSE_LIBTIFF=sys
         -DwxUSE_EXPAT=sys
+        -DwxUSE_GLCANVAS_EGL=OFF
```

Comment:
Can we use `if(CMAKE_SYSTEM_NAME STREQUAL "Linux")` here to make sure this change only works for Linux build?

Reply:
Alright I'll make the change.

Reply:
@SoftFever I've made changes to make this Linux only build logic. Hopefully that works I still don't know what's triggering that wxOut0 issue but apparently it happens just building it normally anyway.